pages/proposals/proposals.jsx

```diff
@@ -206,14 +240,16 @@ var Proposal = React.createClass({
     return this.renderForm();
   },
   render: function() {
+    let stringSource = this.props.localizedStrings;
+
     return (
-      <div className="proposals-page">
+      <div className={classnames(`proposals-page`, this.props.lang.toLowerCase())}>
         <Header/>
         <Jumbotron image="/assets/images/proposals.jpg"
                   image2x="/assets/images/proposals.jpg">
-          <h1>Call for Proposals</h1>
+          <h1>{stringSource.page_banner_header}</h1>
           <div className="deadline">
-            <span>Deadline for submissions is August 1, 2017</span>
+            <span dangerouslySetInnerHTML={{__html: stringSource.page_banner_subheader}}></span>
```

Comment:
can we be certain that this string is always safe? E.g. is there *no possibility* of a user filling in this value?